src/fsharp/vs/IncrementalBuild.fs

```diff
@@ -1367,7 +1367,7 @@ type IncrementalBuilder(ctokCtor: CompilationThreadToken, frameworkTcImportsCach
     let assertNotDisposed() =
         if disposed then  
             System.Diagnostics.Debug.Assert(false, "IncrementalBuild object has already been disposed!")
-    let mutable referenceCount = 0
+    let referenceCount = ref 0
```

Comment:
(Nit: This doesn't need to be a ref - you can take the address of the mutable field)